homeassistant/components/sensor/ebusd.py

```diff
@@ -0,0 +1,188 @@
+"""
+Support for Ebusd daemon for communication with eBUS heating systems.
+For more details about ebusd, please refer to the documentation at
+https://github.com/john30/ebusd
+"""
+
+from datetime import timedelta
+from datetime import datetime
+import logging
+import subprocess
+
+import voluptuous as vol
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+from homeassistant.const import (
+    CONF_NAME, CONF_MONITORED_VARIABLES, STATE_ON, STATE_OFF, STATE_UNKNOWN)
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.entity import Entity
+from homeassistant.util import Throttle
+
+_LOGGER = logging.getLogger(__name__)
+
+DEFAULT_NAME = 'ebusd'
+CONF_CIRCUIT = 'circuit'
+CONF_TIME = 0
```

Comment:
This should be `CONF_TIME = time` as using `0:` in the `configuration.yaml` file is not very intuitive.

Reply:
0 is default value, this is an optional configuration and if not set it will take 0. So it is not needed to specify 0 into `configuration.yaml`